pipelines/fbc-builder/patch.yaml

```diff
@@ -26,3 +26,38 @@
     value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
   - name: PUSH_EXTRA_ARGS
     value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: fbc-validate
+    when:
+    - input: $(params.skip-checks)
+      operator: in
+      values: ["false"]
+    runAfter:
+      - sanity-label-check
+    taskRef:
+      name: fbc-validation
+      version: "0.1"
+    params:
+    - name: IMAGE_URL
+      value: $(tasks.build-container.results.IMAGE_URL)
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: fbc-label-check
+    when:
+    - input: $(params.hacbs)
```

Comment:
hacbs is not here anymore, it should be params.skip-checks in false

Reply:
Just figured that out.
Updating